aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -750,18 +836,19 @@ module aptos_framework::account {
     }
 
     /// create the account for system reserved addresses
-    public(friend) fun create_framework_reserved_account(addr: address): (signer, SignerCapability) {
+    public(friend) fun create_framework_reserved_account(addr: address)
+        : (signer, SignerCapability) {
```

Comment:
Okay, this one was weird.  Ideally `:` should stay with the first line if it's running out
```suggestion
    public(friend) fun create_framework_reserved_account(addr: address): (
        signer, 
        SignerCapability
    ) {
```

Reply:
The downside of this approach is that it's easy to mistake the return type for a function parameter. Although we could change to this kind of output, there are no technical issues.